### PR TITLE
[FIX] History: Bring back the pagination (RT-3255)

### PR DESCRIPTION
**Pull request by vhpoet**

### CODE REVIEW

File: src/jade/tabs/history.jade [hunk 1]
```diff
@@ -194,7 +194,7 @@ section.col-xs-12.content(ng-controller="HistoryCtrl")
                     | Add
                     strong  {{entry.transaction.counterparty | rpcontactname}}
                     |  to contact list
-        nav.paging(ng-if="pagination.count", ng-bind="pagination.pages")
+        nav.paging(ng-if="pagination.count")
```
Comment (mrajvanshy) [inline src/jade/tabs/history.jade]: @vhpoet Do you not have to show the page numbers still? Like {{pagination.pages}}?

Comment (vhpoet) [inline src/jade/tabs/history.jade]: @mrajvanshy nope, that was a mistake to have it there.

**Comment by yongsoo** [comment]
PR to release or develop? @vhpoet

**Comment by mrajvanshy** [comment]
LGTM

**Pull request merged**